Create Call to Action Buttons on Home Page

jacksonmoody opened the pull request:
## Summary

Created call to action buttons highlighting events and communities on the digital.gov home page. 

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/jtm-create-ctas/)

<!--
⚠️ Significant visual changes require submitting previous design to wayback machine.
-->

### Solution

#### Before
<img width="1440" alt="Screenshot 2023-07-14 at 4 17 24 PM" src="https://github.com/GSA/digitalgov.gov/assets/56370370/0bb90451-f426-40ac-968c-b70b23e81635">

#### After
<img width="1440" alt="Screenshot 2023-07-14 at 4 17 45 PM" src="https://github.com/GSA/digitalgov.gov/assets/56370370/0f409586-f9de-4725-b3d4-ccee21eae5d8">

### How To Test

1. Visit the home page
2. Verify that both call to action buttons function correctly

---

### Dev Checklist

- [x] PR has correct labels
- [x] A11y testing (voice over testing, meets WCAG, run axe tools)
- [x] Code is formatted properly

jacksonmoody commented on themes/digital.gov/src/scss/new/_buttons.scss:
Thanks! Addressed in [b81d024](https://github.com/GSA/digitalgov.gov/pull/6317/commits/b81d02472ea354f86d420373299bc4892478bcea)

nick-mon1 reviewed (changes requested):
@jacksonmoody I pushed some style changes for the new buttons to remove the extra spacing mostly for mobile. I also refactored the `margin`, `padding`, and `color` to use uswds tokens. Take a look and feel free to accept or modify.

jacksonmoody commented:
> @jacksonmoody I pushed some style changes for the new buttons to remove the extra spacing mostly for mobile. I also refactored the `margin`, `padding`, and `color` to use uswds tokens. Take a look and feel free to accept or modify.

LGTM, thanks! I just modified to use more native properties, but otherwise should be ready :)

mejiaj reviewed (approved):
Looks good on both small and large screen sizes.

ToniBonittoGSA reviewed (approved):
Nice!
